include/subnet_cadmium.hpp

```diff
@@ -79,14 +119,22 @@ class Subnet {
             }               
         }
 
-        // confluence transition
+        /**
+         * This function calls the internal transition function
+         * followed by external transition function
+         * @param e time variable
+         * @param mbs message bags
+        */
         void confluence_transition(TIME e,
             typename make_message_bags<input_ports>::type mbs) {
             internal_transition();
             external_transition(TIME(), std::move(mbs));
         }
 
-        // output function
+        /**
+         * Function that transmits the message to the output port
```

Comment:
You can consider explaining there will be a 95% probability of transmission of packets and 5% of packet will not be sent. (This is done in the if loop by diving the random number with the maximum random number)

Reply:
solved